configs/centerpoint/centerpoint_0075voxel_second_secfpn_4x8_cyclic_20e_nus.py

```diff
@@ -0,0 +1,206 @@
+_base_ = [
+    '../_base_/models/centerpoint_01voxel_second_secfpn_nus.py',
+    '../_base_/default_runtime.py'
+]
+
+# If point cloud range is changed, the models should also change their point
+# cloud range accordingly
+voxel_size = [0.075, 0.075, 0.2]
+point_cloud_range = [-54, -54, -5.0, 54, 54, 3.0]
```

Comment:
`voxel_size` and `point_cloud_range` are middle variables. We also need to change those keys in model to really modify them.